<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabPoint">
    <attribute name="title">
     <string>Ponto</string>
    </attribute>
    <widget class="QPushButton" name="cancelPointButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="okPointButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxPosPoint">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>240</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Posição</string>
     </property>
     <widget class="QLabel" name="labelXPoint">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="posXPoint">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelYPoint">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="posYPoint">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabLine">
    <attribute name="title">
     <string>Reta</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxP1Line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>240</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Posição Do Ponto 1</string>
     </property>
     <widget class="QLabel" name="labelX1Line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxX1Line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelY1Line">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxY1Line">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="canceLineButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxP2Line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>240</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Posição Do Ponto 2</string>
     </property>
     <widget class="QLabel" name="labelX2Line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelY2Line">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxX2Line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxY2Line">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="okLineButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Polígono</string>
    </attribute>
    <widget class="QPushButton" name="cancelPolygonButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="okPolygonButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxPolyCen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>240</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Posição Do Centro</string>
     </property>
     <widget class="QLabel" name="labelXPolyCen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxXPolyCen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelYPolyCen">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxYPolyCen">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxPolyBorder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>240</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Arestas</string>
     </property>
     <widget class="QLabel" name="labelBordersQtd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Quantidade</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxBordersQtd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelBordersSize">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tamanho</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxBordersSize">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>posXPoint</tabstop>
  <tabstop>posYPoint</tabstop>
  <tabstop>okPointButton</tabstop>
  <tabstop>cancelPointButton</tabstop>
  <tabstop>spinBoxX1Line</tabstop>
  <tabstop>spinBoxY1Line</tabstop>
  <tabstop>spinBoxX2Line</tabstop>
  <tabstop>spinBoxY2Line</tabstop>
  <tabstop>okLineButton</tabstop>
  <tabstop>canceLineButton</tabstop>
  <tabstop>spinBoxXPolyCen</tabstop>
  <tabstop>spinBoxYPolyCen</tabstop>
  <tabstop>spinBoxBordersQtd</tabstop>
  <tabstop>doubleSpinBoxBordersSize</tabstop>
  <tabstop>okPolygonButton</tabstop>
  <tabstop>cancelPolygonButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>canceLineButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
